tar/chunks/chunks_stats_internal.c

```diff
@@ -149,31 +166,30 @@ chunks_stats_print(FILE * stream, struct chunkstats * stats,
 
 	/* Print output line. */
 	if (csv) {
-		if (fprintf(stream,
+		print_field("%s", name, err2);
+		print_sep(",", err2);
 #ifdef STATS_WITH_CHUNKS
-		    "%s,%" PRIu64 ",%s,%s\n",
-		    name, s.nchunks,
-#else
-		    "%s,%s,%s\n",
-		    name,
+		print_field("%12" PRIu64, s.nchunks, err2);
+		print_sep(",", err2);
 #endif
-		    s_lenstr, s_zlenstr) < 0) {
-			warnp("fprintf");
-			goto err2;
-		}
+		print_field("%s", s_lenstr, err2);
+		print_sep(",", err2);
+		print_field("%s", s_zlenstr, err2);
+		print_sep("\n", err2);
 	} else {
-		if (fprintf(stream,
 #ifdef STATS_WITH_CHUNKS
-		    "%-25s  %12" PRIu64 "  %15s  %15s\n",
-		    name, s.nchunks,
+		print_field("%-25s", name, err2);
```

Comment:
I recall that `STATS_WITH_CHUNKS` is old debugging code.

... could I convince you to accept `"%-32"` for the archive name (as we do without that define), instead of `"%-25s"`?  If we're not changing from `-32s` to `-25s`, then the `#if` only needs to cover the two lines referring to the `s.nchunks` field.

Reply:
Yeah that's fine.  I think I had different widths to make it fit into my terminal window when I was writing this back in 2007.